matrix/Matrix.hpp

```diff
@@ -95,7 +95,7 @@ class Matrix
         memcpy(dst, _data, sizeof(Type)*M*N);
     }
 
-    void copyToColumnMajor(Type (&dst)[M*N]) const
+    void copyToColumnMajor(Type dst [M*N]) const
```

Comment:
```suggestion
    void copyToColumnMajor(Type dst[M*N]) const
```
or is this a style preference?